test/javascript/components/toggler.js

```diff
@@ -58,17 +58,31 @@ describe('Toggler', function() {
 
     it('sets aria-expanded to true if the element is visible', function() {
       $html = $('<div id="toggler" data-toggler="class"></div>').appendTo('body');
+      var $triggers = $(`<div>
+          <a data-open="toggler">Open</a>
+          <a data-close="toggler">Close</a>
+          <a data-toggle="toggler">Toggle</a>
+        </div>`).appendTo('body');
```

Comment:
Can you use a generator here to avoid code duplication ?

Reply:
Ok, will prepare it then.